newbank/server/NewBankClientHandler.java

```diff
@@ -58,4 +63,18 @@ public void run() {
     }
   }
 
+  public void requestLoginDetails() {

```

Comment:
Hi Thomas nice job on this.  Only query is should this be a private method?

Reply:
Hi Simon, yes good point, I have added that change to new commit